[2022-06-26 17:39:17,531] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T17:38:57.020911+00:00 [queued]>
[2022-06-26 17:39:17,548] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T17:38:57.020911+00:00 [queued]>
[2022-06-26 17:39:17,549] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 17:39:17,550] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-06-26 17:39:17,550] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-06-26 17:39:17,559] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): postgreSQL_getTable> on 2022-06-26T17:38:57.020911+00:00
[2022-06-26 17:39:17,565] {standard_task_runner.py:52} INFO - Started process 27204 to run task
[2022-06-26 17:39:17,570] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'SDG_Enrique_V0', 'postgreSQL_getTable', '2022-06-26T17:38:57.020911+00:00', '--job-id', '213', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/company_forecasting.py', '--cfg-path', '/tmp/tmpj5ipdn62', '--error-file', '/tmp/tmphf62esjh']
[2022-06-26 17:39:17,575] {standard_task_runner.py:77} INFO - Job 213: Subtask postgreSQL_getTable
[2022-06-26 17:39:17,627] {logging_mixin.py:104} INFO - Running <TaskInstance: SDG_Enrique_V0.postgreSQL_getTable 2022-06-26T17:38:57.020911+00:00 [running]> on host 5e8dc4b25004
[2022-06-26 17:39:17,687] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=enrique-liebana@outlook.com
AIRFLOW_CTX_DAG_OWNER=Enrique
AIRFLOW_CTX_DAG_ID=SDG_Enrique_V0
AIRFLOW_CTX_TASK_ID=postgreSQL_getTable
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T17:38:57.020911+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-26T17:38:57.020911+00:00
[2022-06-26 17:39:17,700] {base.py:74} INFO - Using connection to: id: postgres_conection. Host: host.docker.internal, Port: 5432, Schema: data_company, Login: airflow, Password: XXXXXXXX, extra: None
[2022-06-26 17:39:30,946] {company_forecasting.py:78} INFO - Backup Company data in : /raw_data/company.csv
[2022-06-26 17:39:30,948] {python.py:118} INFO - Done. Returned value was: None
[2022-06-26 17:39:30,959] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=SDG_Enrique_V0, task_id=postgreSQL_getTable, execution_date=20220626T173857, start_date=20220626T173917, end_date=20220626T173930
[2022-06-26 17:39:30,989] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-26 17:39:31,078] {local_task_job.py:146} INFO - Task exited with return code 0
